tools/storage.py

```diff
@@ -1,180 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
no need to worry about it in this PR, but for the future: it is useful in cases like this to start of the PR by moving a file like `tools/storage.py` which shares a lot of the new code to the new location. The reasoning is that it makes navigating back through history via git blame a bit earlier. But again, no big deal here for these little tools.